src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/SolutionCloseTracker.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Razor
+{
+    public abstract class SolutionCloseTracker
+    {
+        public virtual bool IsClosing { get; protected set; }
```

Comment:
Just simplicity, since I wouldn't expect any implementer to add any logic to this. In fact the only reason its virtual is because otherwise Moq doesn't work.